ibis/backends/sqlite/tests/test_client.py

```diff
@@ -75,3 +76,14 @@ def test_connect(url, ext, tmp_path):
     con = ibis.connect(url(path))
     one = ibis.literal(1)
     assert con.execute(one) == 1
+
+
+def test_has_operation(con):
```

Comment:
Arbitrarily decided to test this in the sqlite backend. I think a backend-specific test makes the most sense (since we want to ensure a variety of sources of handling the ops are covered, this kind of needs to be backend-specific).

Reply:
Seems fine to me.